CMakeLists.txt

```diff
@@ -352,7 +352,7 @@ ENDIF()
 SET (Boost_USE_MULTITHREADED ON)
 SET (Boost_FIND_QUIETLY ON)
 
-FIND_PACKAGE (Boost 1.54.0 REQUIRED COMPONENTS date_time regex)
+FIND_PACKAGE (Boost 1.54.0 REQUIRED COMPONENTS date_time regex locale)
```

Comment:
I didn't realize C++11 has a regex implementation.
I chose boost::regex for it's utf8 awareness. The property parser for amounts searches for currency symbols. When using the utf8 aware variant of boost::regex, I can use a character class to filter those. If the same could be easily accomplished with std::regex, I'd happily switch. I haven't found a way to do this in the standard documentation so far.

Note also I didn't add regex as requirement. I added locale.